src/components/IconTooltip/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Theme, Tooltip, TooltipProps } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+import Icon from '../Icon/Icon';
+import { IconName } from '../Icon/icons';
+import { useDeviceInfo } from '../../utils';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  arrow: {
+    color: theme.palette.ClrBaseGray800,
+    marginLeft: '3px',
+    marginTop: '-1px',
+  },
+  icon: {
+    fill: theme.palette.ClrBaseGray800,
+    verticalAlign: 'text-top',
+  },
+  tooltip: {
+    backgroundColor: theme.palette.ClrBaseGray800,
+    borderRadius: '8px',
+    color: theme.palette.ClrBaseWhite,
+    padding: '8px',
+    '& a': {
+      color: theme.palette.ClrBaseWhite,
+    },
+  },
+}));
+
+export type IconTooltipProps = {
+  iconName?: IconName;
+  placement?: TooltipProps['placement'];
+  title: TooltipProps['title'];
+};
+
+export default function IconTooltip({ iconName = 'info', placement = 'top-start', title }: IconTooltipProps): JSX.Element {
+  const { isMobile } = useDeviceInfo();
+  const classes = useStyles();
+
+  return (
+    <Tooltip
+      arrow={true}
+      classes={{
+        arrow: classes.arrow,
+        tooltip: classes.tooltip,
+      }}
+      placement={placement}
+      sx={{
+        maxWidth: isMobile ? '342px' : '464px',
```

Comment:
should the mobile version be a fixed `px` size or can it be relative, e.g. `90%`?

Reply:
Design specs specify:

> max width is 480px on desktop and 358px on mobile (including the tooltip component 8px padding)

These are the resulting values after calculating with **specified value - padding**.

Reply:
But I could see the case for adjusting this. @rachel-tf-design and/or @chudo76 perhaps you may have thoughts on this?

Reply:
I'm going to go ahead and merge this PR, but am happy to update this in a related follow-up PR if we decide we'd like to make the change.